Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -162,9 +176,7 @@ private void RunLoop()
                     {
                         this.chainState.HighestValidatedPoW = this.consensusLoop.Tip;
                         if (this.chain.Tip.HashBlock == block.ChainedBlock?.HashBlock)
-                        {
-                            this.consensusLoop.FlushAsync();
-                        }
+                            await this.consensusLoop.FlushAsync();
```

Comment:
So actually @dangershony and I are thinking that it's a bad idea for this to be async.
Can you please wait for the result of this call and check if it works?

Reply:
OK will fix 👍